crates/matrix-sdk-crypto/src/store/mod.rs

```diff
@@ -887,6 +909,11 @@ impl Store {
             rmp_serde::from_slice(value).map_err(|e| CryptoStoreError::Backend(e.into()))?;
         Ok(deserialized)
     }
+
+    /// Get a stream of updates to inbound group sessions
```

Comment:
I'm trying to avoid the inbound/outbound and group session terms on the public side of the API. Let's stick to the "room key" term like you did for the JS bindings.